### DISCUSSION TITLE
Eclipse Theia v1.48.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13539] [Release Announcements]
Hi everyone 👋,

The Eclipse Theia v1.48.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by jfaltermeier** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by jfaltermeier** [comment]
The release has completed, thank you to everyone that participated and contributed!

**Comment by JonasHelming** [comment]
News & Noteworthy: https://eclipsesource.com/blogs/2024/04/09/eclipse-theia-1-47-release-news-and-noteworthy/
